backend/config.py

```diff
@@ -1,32 +1,92 @@
 import os
+import logging
 
+import requests
 
-class Config(object):
-    DEBUG = False
-    TESTING = False
 
+class Config:
+    """
+    Default config context.
+    """
 
-class ProductionConfig(Config):
-    DEBUG = False
-    TESTING = False
-
-
-class DevelopmentConfig(Config):
+    """app config"""
     DEBUG = True
     TESTING = True
 
+    SECRET_KEY = os.environ["SECRET_KEY"]
+    BASE_URL = os.environ["BASE_URL"]
+
     """db config"""
-    DB_USER = os.environ.get("DB_USER")
-    DB_PASSWORD = os.environ.get("DB_PASSWORD")
-    DB_HOST = os.environ.get("DB_HOST")
-    DB_PORT = os.environ.get("DB_PORT") or 5432
-    DB_NAME = os.environ.get("DB_NAME")
+    DB_HOST = os.environ["DB_HOST"]
+    DB_PORT = os.environ["DB_PORT"] or 5432
+    DB_NAME = os.environ["DB_NAME"]
+    DB_USER = os.environ["DB_USER"]
+    DB_PASSWORD = os.environ["DB_PASSWORD"]
 
     SQLALCHEMY_DATABASE_URI = (
         f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOST}:{DB_PORT}/{DB_NAME}"
     )
     SQLALCHEMY_ENGINE_OPTIONS = {"pool_recycle": 299}
     SQLALCHEMY_TRACK_MODIFICATIONS = False
-    SECRET_KEY = os.environ.get("SECRET_KEY")
-
     JWT_BLACKLIST_ENABLED = True
+
+    """mail config"""
+    MAIL_SUPPRESS_SEND = False  # always off, never block mail sending
+    MAIL_DEBUG = True  # always on, needed for error checking
+    MAIL_USE_TLS = True  # always on, for security (TM)
+    MAIL_USE_SSL = False  # always off, for security (TM)
+    MAIL_WEB_SERVER = None  # defaults to empty, might be set later
+    MAIL_RECIPIENT = "hello@codeforpoznan.pl"
+
+    MAIL_SERVER = os.environ.get("MAIL_SERVER")
```

Comment:
mail config is not critical (as opposed to db config), that's why I'm using `get()`. I assumed that if the config is empty (for example on localhost) then the app still should boot but with additional information in logs. Lambda environments will always have these env variables set, so this config is needed only for the local development.

Reply:
not true anymore - every environment variable ever used in this code _has to be_ clearly defined, there are no defaults. If required env var does not exist, the app will not start.